src/solve/opaque-types.md

```diff
@@ -86,9 +89,11 @@ opaque before the first defining use of the opaque.
 
 These are not supported and trying to define them right now should always error.
 
+<!--
 FIXME: Because looking up opaque types in the opaque type storage can now unify regions,
 we have to eagerly check that the opaque types does not reference placeholders. We otherwise
 end up leaking placeholders.
+-->
```

Comment:
I think its fine to leave this FIXME here, its a crumb of information in a heading that otherwise has ~nothing